rust/core-lib/src/view.rs

```diff
@@ -673,12 +658,7 @@ impl View {
         self.find_changed = false;
 
         self.find.iter().map(|find| {
-            FindStatus {
-                chars: find.search_string().clone(),
-                case_sensitive: Some(find.is_case_sensitive()),
-                is_regex: Some(find.is_regex()),
-                matches: find.occurrences().len(),
-            }
+            find.find_status()
```

Comment:
Not a blocker, but I suspect clippy would nag to write this as `.map(Find::find_status)`.